src/SegmentedComputation.chpl

```diff
@@ -130,54 +129,4 @@ module SegmentedComputation {
     }
     return res;
   }
-
-  proc twoReturnComputeOnSegments(segments: [?D] int, values: [?vD] ?t, param function: SegFunction, type retType) throws {
-    // I hate duplicating the code from computeOnSegments but I need to return 2 arrays
```

Comment:
I don't immediately see any issues with this code but it's def the most complicated/part i understand the least, so it seemed like the natural choice to revert considering the timeout issue